src/wandbot/evaluation/eval/async_main.py

```diff
@@ -46,31 +46,37 @@
 
 
 @retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(6))
-async def get_answer(question: str, application: str = "api-eval-bharat") -> str:
+async def get_answer(
+    question: str, application: str = "api-eval-bharat"
```

Comment:
can you remove your name from default here

Reply:
Updated and changed to `api-eval`